Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+		public static void Demo()
+		{
+			/*2) Take homework 2 code, place it in the forked branch and refactor it using functions.There should be as little duplicate code as possible (there should be functions for:
```

Comment:
Remove such comments from code.

Reply:
removed per instructions. (sorry that was just for me.)